plugin/vimteractive.vim

```diff
@@ -38,12 +41,19 @@ if !has_key(g:, 'vimteractive_default_shells')
 endif
 
 " If 0, disable bracketed paste escape sequences
-let g:vimteractive_bracketed_paste = {
-	\ 'clojure': 0,
-	\ 'python': 0,
-	\ 'python2': 0,
-	\ 'python3': 0,
-	\ }
+if !has_key(g:, 'vimteractive_bracketed_paste')
+	let g:vimteractive_bracketed_paste = { }
+endif
+let g:vimteractive_bracketed_paste.clojure = 0
```

Comment:
I agree this looks much neater

Reply:
It's not only matter of style. If we assign to `g:vimteractive_bracketed_paste` here, it would override settings in `.vimrc`.

Reply:
Agreed